AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -737,72 +737,217 @@ msgstr ""
 "available at: \n"
 "            "
 
-#: AlertaDengue/dados/templates/about.html:149
+#: AlertaDengue/dados/templates/about.html:150
+msgid "Projetos de pesquisa"
+msgstr "Research projects"
+
+#: AlertaDengue/dados/templates/about.html:155
+#, fuzzy
```

Comment:
Be careful with the fuzzy before the **messageId**, you will have to remove the added comments to be able to view the translations, as the content review is done here on github.
https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html